File: Python/8 kyu/simple calculator.py
def calculator(x,y,op):
    
    if op not in ["+", "-", "*", "/"] or type(x) != int or type(y) != int:
        return "unknown value"
    
    calculate = {
        '+': x + y,
        '-': x - y,
        '*': x * y,
        '/': x / y
    }
    
    return calculate[op]

File: JavaScript/8 kyu/simple calculator.js
function calculator(a,b,sign){
    const invalidSign = !['+', '-', '*', '/'].includes(sign);
    const invalidAB = [a, b].some(x => typeof(x) !== 'number');
    
    if (invalidSign || invalidAB) {
      return 'unknown value';
    };
    
    return eval([a, sign, b].join(''));
  }
  